src/tests/multihost/docs/running.rst

```diff
@@ -136,7 +133,7 @@ Executing Tests
     $ py.test --multihost-config=<multihost-template> <test-suite-directory>
 
 * Before executing any tests, it's required to create a config file
-  as specified in `config` section.
+  as specified in `Configuration` section.
 
```

Comment:
I think this may be a good place to put removed `Running Tests` label.

Reply:
There is already an "Executing Tests" title several lines above, but it is probably left out of the _diff_ because it was not modified. I guess you need to check the whole file, not just the differences.

Is that what you were requesting?